tests/conftest.py

```diff
@@ -0,0 +1,23 @@
+from unittest.mock import Mock
+
+import pytest
+
+
+@pytest.fixture
+def mocked_serializer_field():
+    return Mock(context={})
+
+
+@pytest.fixture
+def mocked_serializer_field_with_request_context():
```

Comment:
as I remember you can use `mocker` fixture and mocker.Mock

Reply:
not by default

but i've added pytest-mock as test dependency